### DISCUSSION TITLE
proof of academic status

### DISCUSSION THREAD

**Discussion by Techy5042069** [community/community#36935] [GitHub Education]
My ID card has a different system of date called Bikram Sambat, which might be the case why my request was not approved.
I talked with my high school today to make a 'proof of academic status' but they said , they require the format. 
help me..

**Comment by kraack-tech** [comment]
You could potentially provide your 'offer letter' from the institution, provided it has different date system. Otherwise, you could manually submit an enquiry [here](https://support.github.com/contact/education) regarding your request.
